packages/Operators/src/Search/DTK_ParallelSearch.cpp

```diff
@@ -42,6 +42,7 @@
 #include "DTK_DBC.hpp"
 
 #include <Tpetra_Distributor.hpp>
+#include <Teuchos_Comm.hpp>
```

Comment:
I was using ```allReduce``` for the check but  then I realized it was not necessary but I forgot to remove it.